2D Simulation/initBlocks.js

```diff
@@ -1,88 +1,150 @@
-initTaskID();
-blockColors = initColors(color, NumBlocks);
-flipColorArray = initColors(color, NumBlocks);
-blockLetters = initLetters(letters, NumBlocks);
-flipLetterArray = initLetters(letters, NumBlocks);
-currentConfig = getInitialConfiguration(blockColors, flipColorArray, blockLetters, flipLetterArray);
-
-initBlocks(currentConfig);
-writeBlockStyle(blockColors);
-finalBlocks = getFinalConfiguration(currentConfig);
-
-initInstructions();
-setTaskHeader();
-setIntroduction(1);
-popUpGameIntro();
-
-// function initBlocks(bColors, flipColors, bLetters, flipLtrs) {
-function initBlocks(config) {
-    let bColors = config.map(x => x.topColor);
-    let flipColors = config.map(x => x.bottomColor);
-    let bLetters = config.map(x => x.topLetter);
-    let flipLtrs = config.map(x => x.bottomLetter);
-
-    console.log('Calling initBlocks');
-    // Adds 1 to NumBlocks if the task is matching, ensuring an even
-    // number of blocks to be matched.
-    if (taskID == 1) {
-        if (NumBlocks % 2 != 0) {
-            NumBlocks++;
-        }
+function initAll() {
+    let possibleLetters = ["A", "B", "C", "D", "E", "F", "G"];
+    let possibleColors = ['red', 'blue','green','orange','yellow'];
+
+    if (!isFixed()) {
+        currentConfig = getInitialConfiguration(possibleColors, possibleLetters, NumBlocks);
+        finalBlocks = getFinalConfiguration(currentConfig);
+    } else {
+        // Debug mode (Issue #61)
```

Comment:
I'm looking at issue 61 and this still confuses me a little bit. Shouldn't the E2E tests not rely on some debug mode?

Reply:
The debug mode is going to make it so that the configuration is fixed instead of randomized. That way the E2E tests can be more deterministic.

Perhaps Debug mode was a poor choice of words...